### PR TITLE
Improve Ohai resource docs

### PR DESCRIPTION
**Pull request by tas50**
Improve the examples
Document the action

Signed-off-by: Tim Smith <tsmith@chef.io>

### CODE REVIEW

**Comment by tas50** [comment]
Fixes https://github.com/chef/chef-web-docs/issues/2649

File: content/resources/ohai/_index.md [hunk 1]
```diff
@@ -1,75 +1,88 @@
...
+    markdown: Reload Ohai Data
+properties_list:
+- property: plugin
+  ruby_type: String
+  required: false
+  description_list:
+  - markdown: Specific Ohai attribute data to reload. This property behaves similar
+      to specifying attributes when running Ohai on the command line and takes the
+      attribute that you wish to reload instead of the actual plugin name. For instance
+      you you can pass `ipaddress` to reload `node['ipaddress']` even though that
```
Comment (IanMadd) [inline content/resources/ohai/_index.md]:
```suggestion
      you can pass `ipaddress` to reload `node['ipaddress']` even though that
```

File: content/resources/ohai/_index.md [hunk 2]
```diff
@@ -1,75 +1,88 @@
...
+- "`ohai` is the resource."
+- "`name` is the name given to the resource block."
+- "`action` identifies which steps Chef Infra Client will take to bring the node into
+  the desired state."
+- "`plugin` is the property available to this resource."
+actions_list:
+  :nothing:
+    shortcode: resources_common_actions_nothing.md
+  :reload:
+    markdown: Reload Ohai Data
```
Comment (IanMadd) [inline content/resources/ohai/_index.md]:
```suggestion
    markdown: Reloads the Ohai data.
```

File: content/resources/ohai/_index.md [hunk 3]
```diff
@@ -1,75 +1,88 @@
...
+properties_list:
+- property: plugin
+  ruby_type: String
+  required: false
+  description_list:
+  - markdown: Specific Ohai attribute data to reload. This property behaves similar
+      to specifying attributes when running Ohai on the command line and takes the
+      attribute that you wish to reload instead of the actual plugin name. For instance
```
Comment (IanMadd) [inline content/resources/ohai/_index.md]:
```suggestion
      attribute that you wish to reload instead of the actual plugin name. For instance,
```

**Review by IanMadd — APPROVED** [review]

**Pull request merged**